.github/workflows/check-plugin.yml

```diff
@@ -16,15 +16,23 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
+      - name: Required Packages
+        run: |
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CC86BB64
+          sudo add-apt-repository ppa:rmescandon/yq
+          sudo apt update
+          sudo apt install yq -y
       - name: Checking Availability
         run: |
           #!/bin/bash
           set -e
-          .github/scripts/check_plugin.sh
-          echo ::set-env name=CHIME_MESSAGE::$(cat /tmp/chime_message.md)
+          $GITHUB_WORKSPACE/release-tools/check_plugin.sh
+          echo 'CHIME_MESSAGE<<EOF' >> $GITHUB_ENV
+            cat /tmp/chime_message.md >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
```

Comment:
Why not use one liner code instead of 3?

Reply:
Because when I try this I dont know one liner works.

Reply:
Resolved.